applications/main/nfc/plugins/supported_cards/mykey.c

```diff
@@ -13,7 +13,7 @@ static bool mykey_is_blank(const St25tbData* data) {
 }
 
 static bool mykey_has_lockid(const St25tbData* data) {
-    return (data->blocks[5] & 0xFF) == 0x7F;
+    return (data->blocks[5] >> 24) == 0x7F;
```

Comment:
Can' verify it with my samples. @zProAle do you have documentation supporting this layout?
@augustozanellato can you have a look?

Reply:
> Can' verify it with my samples. @zProAle do you have documentation supporting this layout? @augustozanellato can you have a look?

LGTM, I talked with @zProAle about the issue and suggested him making a PR here. 
The lockid sample I sent you had block5 flipper due to some lack of sleep.

Reply:
Thanks!